Automattic/vip-go-mu-plugins
# Update to JP 8.4.2

## Description

Upgrade to JP 8.4.2

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. `wp plugin list` - ensure JP 8.4.2 is reported
1. WP Admin -> Jetpack - ensure 8.4.2 is reported
